File: Password.py
def check_password(correctpassword):

	for index in range(3):
		user_password = (int(input('Enter security pin (4 digit pin)')))

		if user_password == correctpassword:
			print('Access Granted')	
			break
		else:
			print('Try again')
			

	
check_password(1234)

File: Password.js
const prompt = require('prompt-sync')();
function checkPassword(correctpass){

  
   for(let index = 1; index <= 3; index++){
  
   let userPin = parseInt(prompt("Enter security pin (4 digit pin)"));
   
   if(userPin === correctpass){

      console.log("Access granted");
	return;

    }else{
      
      console.log("Try again");

   }
  }

	console.log("Access denied");
    }
checkPassword(1234);
